start.sh

```diff
@@ -2,17 +2,24 @@
 # start.sh
 
 export LOG_LEVEL="${LOG_LEVEL:-INFO}"
+export PATH="/Users/mitsuhiko/.volta/bin:$PATH"
 
 SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 PROJECT_ROOT="$SCRIPT_DIR"
 
 DIST_SERVER_JS="$PROJECT_ROOT/dist/server.js"
 SRC_SERVER_TS="$PROJECT_ROOT/src/server.ts"
 
+# IMPORTANT: Running from dist/ is strongly recommended
+# There are module resolution issues with tsx/ESM when running from src/ directly
+# If changes are needed, use `npm run build` to compile before running
 if [ -f "$DIST_SERVER_JS" ]; then
-  # echo "INFO: Compiled version found. Running from dist/server.js" >&2 # Silenced
+  echo "INFO: Using compiled version (dist/server.js)" >&2
```

Comment:
printing things in the startup phase confuses some MCP hosts; I'll disable that.

Reply:
Yeah. that was an accidental change that the thing landed.  You can undo these changes.